<launch>
  <node
    pkg="skyrats_cbr_2021"
    type="cross_node"
    name="cross_node" 
    output="screen"
  />
  <node
    pkg="skyrats_cbr_2021"
    type="fase4_bases_fixas.py"
    name="fase1"
    output="screen"
  /> 
  <node
    pkg="skyrats_cbr_2021"
    type="MRS_MAV.py"
    name="MRS_MAV"
    output="screen"
  />
</launch> 
